src/frontend/js/components/CourseProductCourseRuns/EnrolledCourseRun.tsx

```diff
@@ -71,13 +90,28 @@ const EnrolledCourseRun = ({ enrollment }: Props) => {
           </em>
         </div>
         {states.error && <p className="course-runs-list__errors-feedback ">{states.error}</p>}
+
+        {!isStarded && (
+          <p className="course-runs-list__errors-feedback ">
+            <FormattedMessage {...messages.courseRunStartIn} values={{ relativeStartDate }} />
+          </p>
+        )}
         <div className="course-runs-item">
-          <a
-            href={enrollment.course_run.resource_link}
-            className="course-runs-item__cta button--primary button--pill button--tiny"
-          >
-            <FormattedMessage {...messages.goToCourse} />
-          </a>
+          {isStarded && (
+            <a
+              href={enrollment.course_run.resource_link}
+              className="course-runs-item__cta button--primary button--pill button--tiny"
+            >
+              <FormattedMessage {...messages.goToCourse} />
+            </a>
+          )}
+
+          {!isStarded && (
+            <div className="course-runs-item__cta button--primary button--pill button--tiny">
+              <FormattedMessage {...messages.isEnroll} />
+            </div>
+          )}
```

Comment:
Ternary operator is better here nope ?

```suggestion
          {isStarded ? (
            <a
              href={enrollment.course_run.resource_link}
              className="course-runs-item__cta button--primary button--pill button--tiny"
            >
              <FormattedMessage {...messages.goToCourse} />
            </a>
          ) : (
            <div className="course-runs-item__cta button--primary button--pill button--tiny">
              <FormattedMessage {...messages.isEnroll} />
            </div>
          )}
```